### PR TITLE
eras & poeticTypes

### PR DESCRIPTION
**Pull request by esraa898**

### CODE REVIEW

File: resources/views/admin/era/create.blade.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                            <li class="breadcrumb-item active" aria-current="page"><a href="{{URL::current()}}">Add New Era</a></li>
+                        </ol>
+                    </nav>
+
+                </div>
+
+                <div class="container">
+                    <div class="row my-5 mx-auto">
+                        <div class="col-lg-12 layout-spacing">
+                            <form action="{{route('admin.era.store')}}" method="post">
```
Comment (mohamedgouda98) [inline resources/views/admin/era/create.blade.php:37]: the same form used in create file and edit file, so we need to create shard file called "form" and include form file in create and edit

File: resources/views/admin/poeticType/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                            <li class="breadcrumb-item active" aria-current="page"><a href="{{URL::current()}}">Add New PoeticType</a></li>
+                        </ol>
+                    </nav>
+
+                </div>
+
+                <div class="container">
+                    <div class="row my-5 mx-auto">
+                        <div class="col-lg-12 layout-spacing">
+                            <form action="{{route('admin.poeticType.store')}}" method="post">
```
Comment (mohamedgouda98) [inline resources/views/admin/poeticType/create.blade.php:37]: the same form used in create file and edit file, so we need to create shard file called "form" and include form file in create and edit

File: resources/views/admin/era/create.blade.php (hunk 1 above)
Comment (esraa898) [reply resources/views/admin/era/create.blade.php:37]: done

File: resources/views/admin/poeticType/create.blade.php (hunk 2 above)
Comment (esraa898) [reply resources/views/admin/poeticType/create.blade.php:37]: done

**Pull request merged**